angular-front/dist/web-form/en/en/index.html

```diff
@@ -1,14 +0,0 @@
-<!DOCTYPE html><html lang="en" dir="ltr"><head>
```

Comment:
Should I commit this files ?

Reply:
"The dist folder contains the transpiled code of your angular application in the format of JavaScript and also the required html and css files. " Generated after "ng build"